config/routes.rb

```diff
@@ -5,9 +5,18 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'pages#home'
 
-  get 'artists/:id' => 'artists#show', as: :artistid
-  get '/artists/', to: 'artists#all_artists'
-  get 'buyers/:id' => 'buyers#show', as: :buyerid
+  resources :artists, only: [:show] do
+    collection do
+      get 'profile'
```

Comment:
kinda misleading but the `ArtistWorks` component in the artists#show method is really the profile page. can you rename the component and restructure this?